docker/sdk_client.go

```diff
@@ -0,0 +1,37 @@
+package docker
+
+import (
+	"errors"
+	"io/ioutil"
+
+	dlib "github.com/fsouza/go-dockerclient"
+)
+
+var (
+	errNotYetImplemented = errors.New("not yet implemented")
+)
+
+type sdkClient struct {
+	cl *dlib.Client
+}
+
+// NewSDKClient creates a new Client that uses the SDK under the hood
+func NewSDKClient(cl *dlib.Client) Client {
+	return &sdkClient{cl: cl}
+}
+
+func (s *sdkClient) Pull(img *Image) error {
```

Comment:
@mboersma good catch - thanks. Fixed in https://github.com/deis/deisrel/pull/89/commits/f89ec9a1c14bf749095dc43563a00dbd77003aae